README.rst

```diff
@@ -55,65 +72,44 @@ Compatibility Notes
 | Quince           | >= 10.0      |
 +------------------+--------------+
 
-**NOTE**: The Maple version does not support Django 2.2 but it does support Django 3.2 as of eox-core 7.0.
+⚠️ The Maple version does not support Django 2.2 but it does support Django 3.2 as of eox-core 7.0.
 
-The following changes to the plugin settings are necessary. If the release you are looking for is
-not listed, then the accumulation of changes from previous releases is enough.
+The plugin is configured for the latest release (Quince). The following changes in the plugin settings should be applied in order to be used for previous releases.
 
-**Lilac**
-
-.. code-block:: yaml
-
-   EOX_CORE_USERS_BACKEND: "eox_core.edxapp_wrapper.backends.users_l_v1"
-   EOX_CORE_PRE_ENROLLMENT_BACKEND: "eox_core.edxapp_wrapper.backends.pre_enrollment_l_v1"
-   EOX_CORE_ENROLLMENT_BACKEND: "eox_core.edxapp_wrapper.backends.enrollment_l_v1"
+If the release you are looking for is not listed, the actual configuration is sufficient or it is incompatible with the current eox-core version.
```

Comment:
Maybe using _current_ instead of _actual_ is better in this  context? Here is a rewording suggestion:
```suggestion
If the release you are looking for is not listed, then either the current configuration is sufficient, or the release is incompatible with the current eox-core version.
```